package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "space-truckers",
+  "version": "0.0.1",
+  "description": "A game built in BabylonJS. Companion to book",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npx webpack serve --config webpack.dev.js",
+    "build": "npx webpack --config webpack.prod.js",
+    "lint": "npx eslint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jelster/space-truckers.git"
+  },
+  "keywords": [
+    "BabylonJS",
+    "Game"
+  ],
+  "author": "Josh Elster",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/jelster/space-truckers/issues"
+  },
+  "homepage": "https://github.com/jelster/space-truckers#readme",
+  "dependencies": {},
+  "devDependencies": {
+    "@babylonjs/core": "^4.2.0",
```

Comment:
in this kind of project it is pure semantics, but is there a reason not to put the fixed dependencies that are not related to the dev environment in `dependencies`?

Reply:
I was wondering about that too, what sort of difference it makes. I guess I figured that since it's a build-time dep it belonged in devDeps. That emphasizes that it's not a runtime dependency, but you do make a point about it making it more clear what the project is about

Reply:
oh I was thinking of going with this for the versoining: ` @babylonjs/core: "^5.0.0-alpha.9 <= 5.0.0",`